webapp/pages/post/_id/_slug/index.vue

```diff
@@ -84,14 +84,16 @@
       </ds-space>
       <!-- Comments -->
       <ds-section slot="footer">
-        <hc-comment-list
+        <comment-list
           :post="post"
           :routeHash="$route.hash"
           @toggleNewCommentForm="toggleNewCommentForm"
+          @reply="reply"
         />
         <ds-space margin-bottom="large" />
         <comment-form
           v-if="showNewCommentForm && !post.author.blocked"
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-16T10:51:24Z" title="Thursday, January 16th 2020, 11:51:24 am +01:00">Jan 16, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -72,12 +72,18 @@
       <!-- Comments -->
       <ds-section slot="footer">
         <hc-comment-list
+          @reply="reply"
           :post="post"
           :routeHash="$route.hash"
           @toggleNewCommentForm="toggleNewCommentForm"
         />
         <ds-space margin-bottom="large" />
-        <hc-comment-form v-if="showNewCommentForm" :post="post" @createComment="createComment" />
+        <hc-comment-form
+          ref="commentForm"
+          v-if="showNewCommentForm"
```


Maybe put the `v-if` first, to make it more visible: